### PR TITLE
AMP-53: share styles

### PR DESCRIPTION
**Pull request by cole-sanderson**
Setup sharing styles to work with container styles. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-43
 **Linked PRs**: n/a

## Changes
- Add sharing styles in global styles
- Set up styles in container

## How to test-drive this PR
- `npm install`
- Run `npm run dev`
- Preview: http://localhost:3000/
- Make sure that styles are in `<style amp-custom>`

### CODE REVIEW

**Comment by olibrook** [comment]
Looks good to me. I'm just going to mention that people are discussing an amp-first option, in which case we wouldn't be able to have any dependencies on the PWA.

:+1

File: amp/app/styles/global.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// ---
+
+@import 'variables';
+
+
+// Mixins and Functions
+// ---
+
+@import 'lib';
+@import 'node_modules/susy/sass/susy';
```
Comment (kschmidtdev) [inline amp/app/styles/global.scss]: Susy isn't listed in our `package.json` so this fails to build.

File: amp/app/containers/home/container.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@import '../../components/amp-lightbox/theme';
+
+
+// AMP Components
+// ---
+
+// AMP SDK Base Styles from mobify-amp-sdk
+// ---
+
+@import '../../../node_modules/mobify-amp-sdk/src/components/heading/base';
```
Comment (kschmidtdev) [inline amp/app/containers/home/container.scss]: This is referring to a placeholder SDK component - which was just deleted

File: amp/app/containers/home/container.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+// AMP SDK Custom Styles
+// ---
+
+@import '../../styles/themes/amp-components/amp-heading';
```
Comment (kschmidtdev) [inline amp/app/containers/home/container.scss]: This is referring to a placeholder SDK component - which was just deleted

File: amp/app/containers/pdp/container.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+@import '../../components/amp-lightbox/theme';
+
+
+// AMP Components
+// ---
+
+// AMP SDK Base Styles from mobify-amp-sdk
+// ---
+
+@import '../../../node_modules/mobify-amp-sdk/src/components/heading/base';
```
Comment (kschmidtdev) [inline amp/app/containers/pdp/container.scss]: This is referring to a placeholder SDK component - which was just deleted

File: amp/app/containers/pdp/container.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+
+// AMP SDK Custom Styles
+// ---
+
+@import '../../styles/themes/amp-components/amp-heading';
```
Comment (kschmidtdev) [inline amp/app/containers/pdp/container.scss]: This is referring to a placeholder SDK component - which was just deleted

**Pull request merged**